### PR TITLE
ESLint & Prettier in the workflows

### PR DESCRIPTION
**Pull request by bina-biene**
Adds a super simple action to run prettier and ESLint.
* Example of a failing test (didn't run prettier on the configuration file): [here](https://github.com/volunteer-prb/war-agregator/runs/6611649433?check_suite_focus=true).

### CODE REVIEW

**Comment by bina-biene** [comment]
Looks like there are some permission issues preventing the action from getting executed. Action link: [here](https://github.com/volunteer-prb/war-agregator/actions/runs/2380306245). Docs: [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository).

**Comment by AIRTucha** [comment]
I will contact Admins

File: frontend/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "gen:api": "openapi-typescript ../backend/open-api.json --output src/open-api.ts",
+    "gen:api": "openapi-typescript ../backend/open-api.json --output src/api/open-api.ts",
```
Comment (bina-biene) [inline frontend/package.json:24]: Not related to the workflows, I just forgot to update this in the previous PR.

**Review by AIRTucha — APPROVED** [review]
Please squash commits on merge

**Pull request merged**